### PR TITLE
added keys

### PR DESCRIPTION
**Pull request by lkatalin**
Added my key and Mark's key.

### CODE REVIEW

**Review by npmccallum — CHANGES_REQUESTED** [review]
I think we need this format:

```
users.users = {
  npmccallum = { ... };
  mbestavros = { ... };
  lsturman = { ... };
};
```

Also, I don't think I want to grant you root access. So no `wheel` group.

I'll have to add an sgx group and a udev entry for it.

**Comment by lkatalin** [comment]
Thanks - let me know if this new version works.

File: configuration.nix [hunk 1]
```diff
@@ -38,13 +38,28 @@
   security.pam.services.login.makeHomeDir = true;
   security.pam.services.sshd.makeHomeDir = true;
 
-  users.mutableUsers = false;
```
Comment (steveej) [inline configuration.nix:41]: Did you remove this line intentionally?

File: configuration.nix [hunk 2]
```diff
@@ -38,13 +38,28 @@
...
-  users.users.npmccallum = {
-    isNormalUser = true;
-    extraGroups = [ "wheel" ];
-    openssh.authorizedKeys.keys = [
-      "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBIoZP5bZedmrj/lidLkKXhvZwwl9Pj5VxLV22nXhkijt7UJhSUX/rOV4Kg/wmR5ptMjGyE4PPSHmCEzXvQnpyMU= nathaniel@mccallum.life"
-    ];
+  users.users = {
+    npmccallum = {
+      isNormalUser = true;
+      extraGroups = [ "wheel" ];
```
Comment (steveej) [inline configuration.nix]: should probably also be in the sgx group, and only use wheel privileges for other operations.

File: configuration.nix (hunk 1 above)
Comment (lkatalin) [reply configuration.nix:41]: No; good catch. : )

**Review by npmccallum — APPROVED** [review]

**Pull request merged**